### PR TITLE
doc: added section on elektraPluginGetGlobalKeySet to plugin tutorial

### PR DESCRIPTION
**Pull request by vLesk**
Added a section on elektraPluginGetGlobalKeySet to the plugin tutorial as referenced in #2524 

## Basics

Check relevant points but **please do not remove entries**.
Do not describe the purpose of this PR in the PR description but:

- [x] Short descriptions should be in the release notes (added as entry in
      `doc/news/_preparation_next_release.md` which contains `_(my name)_`)
      **Please always add something to the the release notes.**
- [x] Longer descriptions should be in documentation or in design decisions.
- [x] Describe details of how you changed the code in commit messages
      (first line should have `module: short statement` syntax)
- [ ] References to issues, e.g. `close #X`, should be in the commit messages.

## Checklist

Check relevant points but **please do not remove entries**.
For docu fixes, spell checking, and similar none of these points below
need to be checked.

- [ ] I added unit tests
- [ ] I ran all tests locally and everything went fine
- [x] affected documentation is fixed
- [ ] I added code comments, logging, and assertions (see doc/CODING.md)
- [ ] meta data is updated (e.g. README.md of plugins and doc/METADATA.md)

## Review

Remove the line below and add the "work in progress" label if you do
not want the PR to be reviewed:

@markus2330 please review my pull request

## Merging

Please add the "ready to merge" label when the build server and you say
that everything is ready to be merged.

### CODE REVIEW

File: doc/tutorials/plugins.md [hunk 1]
```diff
@@ -441,6 +441,24 @@ Plugin *ELEKTRA_PLUGIN_EXPORT
 
 For further information see [the API documentation](https://doc.libelektra.org/api/current/html/group__plugin.html).
 
+### `elektraPluginGetGlobalKeySet`
+
+In order to enable communication between plugins which is more complex than what can be done with metadata, Elektra provides a global keyset which plugins can read from and modify.
+
+The keyset is initialized by the KDB and can be accessed by all plugins except for plugins created manually (e.g. with `elektraPluginOpen`).
```
Comment (mpranj) [inline doc/tutorials/plugins.md]: I'm not sure if it is important to mention that it is also closed with the KDB handle (`kdbClose`), and data is not shared with other KDB handles (i.e. it is not a mechanism for inter process communication).

File: doc/tutorials/plugins.md [hunk 2]
```diff
@@ -441,6 +441,24 @@ Plugin *ELEKTRA_PLUGIN_EXPORT
...
+
+It can be accessed by calling the following function:
+
+```c
+KeySet * elektraPluginGetGlobalKeySet (Plugin * plugin);
+```
+
+The handle of the plugin using the keyset needs to be provided as `plugin`.
```
Comment (mpranj) [inline doc/tutorials/plugins.md]: I don't think this has to be mentioned (not here). Passing this `Plugin *` is used in all exported functions of plugins and lots of other places. If this info is missing, maybe mention it somewhere in a more general section.

File: doc/tutorials/plugins.md (hunk 1 above)
Comment (markus2330) [reply doc/tutorials/plugins.md]: Yes, sounds also useful to me to mention this here.

**Comment by mpranj** [comment]
Thank you! I think you will need to update your fork and rebase from master to fix the broken link issues in the tests.
Otherwise it looks good to me!

**Comment by markus2330** [comment]
Thank you, great work! I am looking forward to see more.

**Pull request merged**